warp.properties

```diff
@@ -0,0 +1,7 @@
+wd.warp.jdbc.url=jdbc:mysql://localhost:3307/warp_core?createDatabaseIfNotExist=true&zeroDateTimeBehavior=convertToNull&verifyServerCertificate=false&useSSL=true
```

Comment:
do we want to add back the `do not use in production` comment?

Reply:
that comment exists in docker-compose file. ill add it here too